### PR TITLE
Map LeavesFix

### PR DESCRIPTION
**Pull request by Martmists-GH**

### CODE REVIEW

File: mappings/net/minecraft/datafixer/fix/LeavesFix.mapping [hunk 1]
```diff
@@ -1,4 +1,22 @@
 CLASS net/minecraft/class_1191 net/minecraft/datafixer/fix/LeavesFix
+	FIELD field_5686 logBlocks Ljava/util/Set;
+	FIELD field_5688 leafMap Lit/unimi/dsi/fastutil/objects/Object2IntMap;
```
Comment (liach) [inline mappings/net/minecraft/datafixer/fix/LeavesFix.mapping]: static final fields should use upper camel case, like `LEAVES_MAP`

File: mappings/net/minecraft/datafixer/fix/LeavesFix.mapping [hunk 2]
```diff
@@ -1,4 +1,22 @@
...
+	CLASS class_1192
```
Comment (liach) [inline mappings/net/minecraft/datafixer/fix/LeavesFix.mapping]: You probably should name this class.

Comment (Martmists-GH) [reply mappings/net/minecraft/datafixer/fix/LeavesFix.mapping]: Honestly no clue what would be a proper name

File: mappings/net/minecraft/datafixer/fix/LeavesFix.mapping (hunk 1 above)
Comment (Martmists-GH) [reply mappings/net/minecraft/datafixer/fix/LeavesFix.mapping]: Whoops, missed that

File: mappings/net/minecraft/datafixer/fix/LeavesFix.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/datafixer/fix/LeavesFix.mapping]: Looks like a `LeavesLogFixer` or such

File: mappings/net/minecraft/datafixer/fix/LeavesFix.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/datafixer/fix/LeavesFix.mapping]: (same goes for `LOG_BLOCKS` etc)

**Review by Runemoro — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Pull request merged**